client/src/partials/admin/transaction_type/transaction_type.js

```diff
@@ -20,10 +20,14 @@ function TransactionTypeController(TransactionType, TransactionTypeStore, Notify
 
   // edit button template
   var editTemplate = '<div class="ui-grid-cell-contents">' +
-    '<a title="{{ \'FORM.LABELS.EDIT\' | translate }}" href="" ' +
+    '<a href="" title="{{ \'FORM.LABELS.EDIT\' | translate }}" ' +
     'ng-click="grid.appScope.editType(row.entity)" ' +
+    'uib-popover="{{grid.appScope.notAllowed(row.entity.fixed) | translate }}" ' +
```

Comment:
![transaction-type](https://cloud.githubusercontent.com/assets/5445251/18477739/95eb2880-7983-11e6-887e-bb9cd64b60c2.jpg)

If the row is not fixed, the popover doesn't appear